examples/react-hooks/next/components/Menu.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Link from 'next/link';
+
+export default function Menu() {
+  return (
+    <div>
+      Menu:
+      <ul style={{ display: 'flex', gap: '1em', listStyle: 'none' }}>
+        <li>
+          <Link href={`/search/`}>Home</Link>
```

Comment:
the right way to make this would be `useMenu` + Portal, no? Again, this is good for what people asked for in the original question, but I don't think for the main demo

Reply:
Yeah I'll make an easier dynamic routing example in a separate demo for routing showcase

Reply:
Okay I made an example with navigation to another route without InstantSearch, and to show that Next.js links work I just also put a `Link` to show that navigation links outside of InstantSearch do update the state as well